File: Clases-Bluej/Sistema_ticketing/Cliente.java
import java.util.ArrayList;

public class Cliente {
    private String idCliente;
    private String nombre;
    private String telefono;
    private String direccion;
    private ArrayList<Mascota> mascotas;

    public Cliente(String idCliente, String nombre, String telefono, String direccion) {
        this.idCliente = idCliente;
        this.nombre = nombre;
        this.telefono = telefono;
        this.direccion = direccion;
        this.mascotas = new ArrayList<>();
    }

    public void agregarMascota(Mascota mascota) {
        this.mascotas.add(mascota);
    }

    public String getIdCliente() { return idCliente; }
    public String getNombre() { return nombre; }
    public String getTelefono() { return telefono; }
    public String getDireccion() { return direccion; }
    public ArrayList<Mascota> getMascotas() { return mascotas; }
}

File: Clases_Csharp/Cliente.cs
using System.Collections.Generic;

namespace SistemaVeterinaria
{
    public class Cliente
    {
        public string IdCliente { get; private set; }
        public string Nombre { get; private set; }
        public string Telefono { get; private set; }
        public string Direccion { get; private set; }
        public List<Mascota> Mascotas { get; private set; }

        public Cliente(string idCliente, string nombre, string telefono, string direccion)
        {
            IdCliente = idCliente;
            Nombre = nombre;
            Telefono = telefono;
            Direccion = direccion;
            Mascotas = new List<Mascota>();
        }

        public void AgregarMascota(Mascota mascota)
        {
            Mascotas.Add(mascota);
        }
    }
}